### ISSUE TITLE
TypeError: Cannot read property 'marker' of undefined

### ISSUE THREAD

**Issue by bradgarropy** [andrewbranch/gatsby-remark-vscode#93]
After updating to `v2.1.0` I am now getting this error.

My `gatsby-config.js` file looks like this:

```javascript
                    {
                        resolve: "gatsby-remark-vscode",
                        options: {
                            theme: "Shades of Purple",
                            extensions: ["shades-of-purple"],
                        },
                    },
```

I have not defined the `inlineCode` property at all.

The `markdown` I'm trying to render looks like this:

```markdown
Next we need to actually render an image in the slider. Use a CSS `background-image` property so you can render the image. You should also create a reference to this element to use later when we modify what image is displayed.
```

For reference, here is the repository where this is happening:
https://github.com/bradgarropy/bradgarropy.com/tree/channel

**Comment by bradgarropy** [comment]
Looks like there might need to be a check here to see if `inlineCode` exists first.
https://github.com/andrewbranch/gatsby-remark-vscode/blob/e4a237feec4093cd102e12aec3002ee0889c3d09/src/index.js#L90

**Comment by andrewbranch** [comment]
Good catch, thank you! That’s a missing test case.

**Comment by andrewbranch** [comment]
Published as v2.1.1. Thanks again!

**Comment by bradgarropy** [comment]
Really appreciate the super fast turnaround, thank you! 🙏🏼

`v2.1.1` is now in production on [bradgarropy.com](https://bradgarropy.com)!
